content/departments/engineering/dev/process/pull-requests.md

```diff
@@ -0,0 +1,115 @@
+# Pull requests
+
+This guide will walk you through the process of creating, reviewing, and approving pull requests (PRs). It covers the following topics:
+
+1. How to create a PR
+2. How to make the PR easy to review
+3. How to review PRs
+4. What "quick stamp"s mean
+
+## 1. How to create a PR
```

Comment:
This section is the old docs, I didn't change it (okay, maybe I did 1% formatting, but no real changes)